crown/src/utils/slogger.rs

```diff
@@ -8,6 +8,7 @@ use sword::mem::NockStack;
 use sword::noun::{Atom, DirectAtom, IndirectAtom, Noun, Slots};
 use sword_macros::tas;
 
+#[allow(unused_imports)]
```

Comment:
@bitemyapp this shouldn't be necessary if the feature-flagging is working properly, right?

Reply:
even with this change seems like CI is still failing anyway